### ISSUE TITLE
The advantage in V1 version

### ISSUE THREAD

**Issue by yiwei0730** [SWivid/F5-TTS#868]
### Checks

- [x] This template is only for question, not feature requests or bug reports.
- [x] I have thoroughly reviewed the project documentation and read the related paper(s).
- [x] I have searched for existing issues, including closed ones, no similar questions.
- [x] I confirm that I am using English to submit this report in order to facilitate communication.

### Question details

ZH_version:
我在發布版本更新中有看到了你提到的更改
我想知道這兩個版本的變化有什麼區別，以及優點和缺點。
因為我聽了demo好像沒什麼太大的差別，主要是1.0版本的聲音不會有太大的變化，但是音質好像稍微差了一點。 0.6 會有些不同。
謝謝
EN_version:
I saw the changes you mentioned in the release version update
I want to know the differences between these changes in the two versions, as well as the advantages and disadvantages. 
Because I listened to the demo and there doesn't seem to be much difference, mainly because the sound of version 1.0 will not change much, but the sound quality seems to be slightly worse. 0.6 will be a little different.
Thank you.

**Comment by SWivid** [comment]
> Because I listened to the demo and there doesn't seem to be much difference, mainly because the sound of version 1.0 will not change much, but the sound quality seems to be slightly worse. 0.6 will be a little different.

The speaker similarity has improved (accent, timbre), previous version zh-zero-shot-example gives a standard mandarin output while v1 gives more sichuan accent, right?

**Comment by ruby11dog** [comment]
> > Because I listened to the demo and there doesn't seem to be much difference, mainly because the sound of version 1.0 will not change much, but the sound quality seems to be slightly worse. 0.6 will be a little different.
> 
> The speaker similarity has improved (accent, timbre), previous version zh-zero-shot-example gives a standard mandarin output while v1 gives more sichuan accent, right?

What's the reaseon V1 can bring more accent?

**Comment by SWivid** [comment]
> What's the reaseon V1 can bring more accent?

@ruby11dog better speech-text modality alignment modeling ability leaves more learning space for other aspects
All three fix contribute to that, WER decreases fast so model can take care more SIM and stuff (prosody e.g. from text format instruct

**Comment by ruby11dog** [comment]
> > What's the reaseon V1 can bring more accent?
> 
> [@ruby11dog](https://github.com/ruby11dog) better speech-text modality alignment modeling ability leaves more learning space for other aspects All three fix contribute to that, WER decreases fast so model can take care more SIM and stuff (prosody e.g. from text format instruct

thx for reply, besides, what is the advantage of the updates:"zero-initialize adaptive layernorm in DiT"

**Comment by SWivid** [comment]
> what is the advantage of the updates:"zero-initialize adaptive layernorm in DiT"

@ruby11dog https://arxiv.org/pdf/2212.09748 page 6 right bottom
in our expr: faster convergence, a bit improvement on WER on long round, removing some glitches artifact as normalized better

**Comment by SWivid** [comment]
@SynthAether wish good result.
There's one thing might be useful, need to ensure the dataset is a fully 24khz one, or it would be better to resample to 16khz and train a 16khz model. 
In our case, as Emilia has many not fully 24khz audio samples (which is our train set), when inference with 16khz prompt, will have droplet artifact in high freq.

Another way is to turn on qknorm, but would train and inference slower.
https://github.com/SWivid/F5-TTS/blob/1ab90505a4a79a27f6e3b237c021429a1e6c7e9b/src/f5_tts/configs/F5TTS_v1_Base.yaml#L32

**Comment by SynthAether** [comment]
@SWivid Thanks for the tip, much appreciated. We trained a small model, `F5TTS_Small`, on our own 22050 Hz data and some external dataset (but not on Emelia dataset), we maintained a native sampling rate of 24 kHz so that upsampling occurs during training from 22.05 kHz to 24 kHz, this way we can leverage the pretrained `vocos` model. Our prompts consisted of samples at or above 22.05 kHz. We incorporated multiple languages with a heavy emphasis on English and other Western languages, dedicating 500 hours to a couple of the main languages.

After training for 600K steps on 8 older GPUs, the loss has remained within acceptable bounds, no `NAN` and no instabilities, and the loss is ranging from 0.59 to 0.9 (mostly hovers around 0.6), and the audio quality is promising. However, we still experience issues such as word dropping and repetition, or hearing last few words from the reference input template. We experimented with different `fix_duration` and `target_rms` values during inference, as well as adjusting our templates and text prompts, which helped in certain cases. The longer we train though, the more we notice continuous improvements in WER reduction. Although it is hard to predict exactly when we will achieve a satisfactory WER, these trends suggest that extended training will yield better results. My main concern is that 500 hours of speech per language may not be sufficient, and I am now eager to stop this training run and switch to the latest code with `V1`, albeit a smaller variant of `V1`.

**Comment by ruby11dog** [comment]
v1 model'sconvergence speed is decent in my experiment 😄

**Comment by rlenain** [comment]
@SWivid 
are there modeling differences between base F5-TTS and v1 F5-TTS? Or is it only that you have managed to train it better? If there are modeling differences, do you intend on releasing a paper with some details? Thank you.

**Comment by SWivid** [comment]
@rlenain could check the release note.
mainly fix of implementation, so we named it as v1

**Comment by dinhthuantel4vn** [comment]
> In the published English demo, the audio produced by V1 sounds much closer to the template wav file. I look forward to training V1 on 8 GPUs (40 GB each). To avoid reducing the values for `batch_size_per_gpu` or `max_samples`, I'll use a smaller model configuration, e.g. `dim`=768, `depth`=18, `heads`=12

Hey bro, can you share the trained checkpoint? I’ve been searching for an F5TTS small checkpoint but haven’t found one.

**Comment by S-T-K** [comment]
@SWivid Is the inference speed expected to be the roughly same for 0.6 and 1.0?

**Comment by Ming-er** [comment]
@SWivid Hi, authors. Are there any quantitative results of the new version?

**Comment by SWivid** [comment]
> Is the inference speed expected to be roughly the same for 0.6 and 1.0?

@S-T-K yes, roughly the same

> Are there any quantitative results of the new version?

@Ming-er
seed 0,1,2 avg
ls-pc wer: 2.33%->2.09%; sim: 0.659->0.673
seed-en wer: 1.76%->1.55%; sim: 0.664->0.679
etc. 
feel free to test
